Server/ReasnAPI/ReasnAPI/Services/Authentication/AuthService.cs

```diff
@@ -68,6 +68,8 @@ public User Register(RegisterRequest request)
                 City = request.Address.City,
                 Street = request.Address.Street,
                 State = request.Address.State,
+                ZipCode = !string.IsNullOrEmpty(request.Address.ZipCode) ?
+                    request.Address.ZipCode : null
```

Comment:
```suggestion
                ZipCode = request?.Address?.ZipCode
```

Reply:
Context: we can use null propagation to check for nulls, for empty ones I think we don't care really, propably it will be handled on FE anyways

Reply:
But if you really want to check it for being empty then alright, let's go with that

Reply:
Turned out that not existing fields are set to null by default, however if their value will be set to empty string this violates database and causing exception